lib/matplotlib/image.py

```diff
@@ -1547,7 +1547,15 @@ def imread(fname, format=None):
 def imsave(fname, arr, vmin=None, vmax=None, cmap=None, format=None,
            origin=None, dpi=100, *, metadata=None, pil_kwargs=None):
     """
-    Save an array as an image file.
+    Colormap and save an array as an image file.
+
+    RGB(A) images are passed through, single channel images will be
+    saved in false color.
```

Comment:
"will be colormapped"? "will be colormapped according to *cmap* and *norm*"?

Reply:
I thought "colormapped" was too jargonny, but I like "will be colormapped according to cmap and norm".